aiocloudweather/station.py

```diff
@@ -90,13 +90,13 @@ class WundergroundRawSensor:
     windgustdirection: float = field(
         default=None, metadata={"unit": DEGREE, "arg": "windgustdir"}
     )
-    uv: int = field(default=None, metadata={"unit": UV_INDEX, "arg": "UV"})
+    uv: int = field(default=None, metadata={"unit": UV_INDEX, "arg": "UV"}
+    )
     solarradiation: float = field(
         default=None,
         metadata={
-            "unit": LIGHT_LUX,
-            "factor": 1000,
-            "arg": "solarRadiation",
+            "unit": UnitOfIrradiance.WATTS_PER_SQUARE_METER,
+            "arg": "solarradiation",
```

Comment:
This will break the functionality for stations which still use the old parameter `solarRadiation` and LUX instead of W/m². I am currently working on providing an alternative PR. Just need to do some testing. Including your line from the PR.